cmake/modules/FindPySpark.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#
+# Try to find pyspark
+# Once done this will define
+#
+#  PYSPARK_FOUND - system has pyspark and its dependencies are met
+#
+
+message(STATUS "Looking for pyspark dependency: Java")
```

Comment:
Shouldn't this define the function `findPySpark` (or whatever the CMake convention is)?

Reply:
I read a bit more about this since I'm not expert and Oksana gave me some very helpful suggestions. The CMake convention is to have a `FindX.cmake` file define a set of variables that flag the availability of the package in the system (for example the home directory of the package) and then use those as arguments to [find_package_handle_standard_args](https://cmake.org/cmake/help/v3.9/module/FindPackageHandleStandardArgs.html) . Examples we use in ROOT with python modules are in [FindNumpy.cmake](https://github.com/root-project/root/blob/a56901942dfefa6e7e2e7df397a60c9d1cda31a0/cmake/modules/FindNumPy.cmake#L100-L103) and the function [find_python_module](https://github.com/root-project/root/blob/a56901942dfefa6e7e2e7df397a60c9d1cda31a0/cmake/modules/RootMacros.cmake#L1908-L1909)